File: examples/hello.js
/**
 * 帮助示例 (Help Example)
 * 
 * 这是一个简单的 JavaScript 示例，展示如何创建有用的函数。
 * This is a simple JavaScript example showing how to create useful functions.
 */

// 问候函数 (Greeting function)
function helloWorld() {
    console.log("你好，世界！(Hello, World!)");
    console.log("欢迎使用帮助项目！(Welcome to the Help Project!)");
}

// 帮助信息函数 (Help information function)
function helpMessage() {
    const helpText = `
    帮助项目示例 (Help Project Example)
    
    这个脚本演示了基本的 JavaScript 功能：
    This script demonstrates basic JavaScript functionality:
    
    - 函数定义 (Function definition)
    - 控制台输出 (Console output)
    - 模板字符串 (Template strings)
    - 模块导出 (Module exports)
    
    使用方法 (Usage):
    node examples/hello.js
    `;
    console.log(helpText);
}

// 用户交互函数 (User interaction function)
function greetUser(name = "朋友 (Friend)") {
    console.log(`你好，${name}！很高兴认识你！`);
    console.log(`Hello, ${name}! Nice to meet you!`);
}

// 主函数 (Main function)
function main() {
    console.log("=".repeat(50));
    helloWorld();
    console.log("=".repeat(50));
    helpMessage();
    console.log("=".repeat(50));
    greetUser("开发者 (Developer)");
    console.log("=".repeat(50));
}

// 如果直接运行此文件 (If running this file directly)
if (require.main === module) {
    main();
}

// 导出函数以供其他模块使用 (Export functions for use by other modules)
module.exports = {
    helloWorld,
    helpMessage,
    greetUser,
    main
};

File: examples/hello.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
帮助示例 (Help Example)

这是一个简单的 Python 示例，展示如何创建一个有用的脚本。
This is a simple Python example showing how to create a useful script.
"""

def hello_world():
    """
    打印问候语的函数
    Function to print greetings
    """
    print("你好，世界！(Hello, World!)")
    print("欢迎使用帮助项目！(Welcome to the Help Project!)")

def help_message():
    """
    显示帮助信息
    Display help information
    """
    help_text = """
    帮助项目示例 (Help Project Example)
    
    这个脚本演示了基本的 Python 功能：
    This script demonstrates basic Python functionality:
    
    - 函数定义 (Function definition)
    - 字符串处理 (String handling)
    - 用户交互 (User interaction)
    
    使用方法 (Usage):
    python examples/hello.py
    """
    print(help_text)

def main():
    """
    主函数
    Main function
    """
    print("=" * 50)
    hello_world()
    print("=" * 50)
    help_message()
    print("=" * 50)
    
    # 用户交互示例 (User interaction example)
    user_input = input("请输入你的名字 (Please enter your name): ")
    print(f"你好，{user_input}！很高兴认识你！")
    print(f"Hello, {user_input}! Nice to meet you!")

if __name__ == "__main__":
    main()